src/modules/flow/lcd/Kconfig

```diff
@@ -0,0 +1,4 @@
+config FLOW_NODE_TYPE_LCD
+	tristate "Node type: lcd"
+	depends on USE_I2C
```

Comment:
not in the same module, as we faced the issue with grove, it would happen for other modules as well :-)

You can alias to that, but not call the module after that. Call the module after the chip.